C-Plus-Plus/cp/Minimum_steps_to_collect_coin_stacks.cpp

```diff
@@ -0,0 +1,72 @@
+/*Problem Statement: “Given an integer arrays representing the height of each stack of coins and the number of coin stacks, calculate the minimum number of straight lines that pass through all the coins (minimum number of steps to collect these coins).”*/
+
+
+// Hints: Divide and Conquer
+#include <iostream>
```

Comment:
Use **bits/stdc++**

Reply:
I've added all the changes as requested in PR #7507. Please review asap and merge if all's well!